### DISCUSSION TITLE
`日志` or `日记`?

### DISCUSSION THREAD

**Discussion by 3gf8jv4dv** [libnyanpasu/clash-nyanpasu#1764] [General]
I noticed that the term *log* uses different Chinese translations in different places in the project.

- Software: 日志
- Website & Issue Templates: 日记

Generally, the vast majority of projects I have seen use *日志*, and almost no *日记*. I would like to know if this is intentional or a mistake?

**Comment by keiko233** [comment]
Word we probably use most often is 日志, and 日记 has a similar meaning. According to habits of software development, 日志 may be more appropriate.
